invenio_vocabularies/contrib/subjects/record_type.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we are calling this file `Affiliations.py`, so either we call this `subjects.py` or we rename affiliations. I'm leaning towards `subjects.py`

Reply:
Saw the pre-existing `subjects.py` I guess it was done like this to avoid removing the old one. I move it to `subjects.py`.